### PR TITLE
feat(bigquery): add notification method to bigquery client and logic …

### PR DESCRIPTION
**Pull request by kapoc3**
1. add notification method to bigquery client 
2. add logic for select emails to notificate
3. update test suit including notify mock

### CODE REVIEW

File: src/components/Integrations/BigQuery/AuthorizationPage.js [hunk 1]
```diff
@@ -31,13 +31,24 @@ const AuthorizationLayout = ({ dependencies: { bigQueryClient, dopplerUserApiCli
     fetchData();
   }, [bigQueryClient, dopplerUserApiClient]);
 
-  const onSubmit = async (values) => {
+  const onSubmit = async (values, emailsToNotify) => {
+    debugger;
```
Comment (jcamposmk) [inline src/components/Integrations/BigQuery/AuthorizationPage.js:35]: you should remove all debugger

File: src/components/Integrations/BigQuery/ControlPanel/AuthorizationForm.test.js [hunk 2]
```diff
@@ -1,70 +1,60 @@
 import React from 'react';
-import { render, fireEvent, waitFor, screen } from '@testing-library/react';
+import { render, fireEvent, waitFor, screen, act } from '@testing-library/react';
 import '@testing-library/jest-dom/extend-expect';
 import IntlProvider from '../../../../i18n/DopplerIntlProvider.double-with-ids-as-values';
 import { AuthorizationForm } from './AuthorizationForm';
-
+import user from '@testing-library/user-event';
```
Comment (jcamposmk) [inline src/components/Integrations/BigQuery/ControlPanel/AuthorizationForm.test.js:6]: You are importing userEvent, but you are not using it!
Could you change fireEvent to userEvent? It's very easy. 

change method: 
`await fireEvent.change(input, { target: { value: tagToAdd1 } });  -> await userEvent.type(input, tagToAdd1);`

click method: 
`fireEvent.click(addButton); -> userEvent.click(addButton);`

Enter method: 
`fireEvent.keyDown(input, { key: 'Enter', code: 'Enter' }); -> userEvent.type(input, '{enter'});`

Comment (jcamposmk) [reply src/components/Integrations/BigQuery/ControlPanel/AuthorizationForm.test.js:6]: Add spaces
image

Comment (kapoc3) [reply src/components/Integrations/BigQuery/ControlPanel/AuthorizationForm.test.js:6]: ok i agree, in this ocation the import was used for test prupose, i wll to remove this import

File: src/components/Integrations/BigQuery/ControlPanel/AuthorizationForm.test.js [hunk 3]
```diff
@@ -83,7 +72,6 @@ describe('AuthorizationForm ', () => {
     // the same amount of tag is kept because it was not added
     expect(cloudTags.querySelectorAll('li').length).toBe(emails.length + 1);
     expect(errors).toBeInTheDocument();
-
     // success when add second tag (simulated with enter event)
     await fireEvent.change(input, { target: { value: tagToAdd2 } });
```
Comment (jcamposmk) [inline src/components/Integrations/BigQuery/ControlPanel/AuthorizationForm.test.js:76]: In this part, you should use userEvent.clear(input) before userEvent.type method. This for clean input field before typing. 
image

File: src/components/Integrations/BigQuery/AuthorizationPage.test.js [hunk 4]
```diff
@@ -10,30 +10,37 @@ describe('test for validate authorization form component ', () => {
     emails: ['email1@gmail.com', 'email2@gmail.com', 'email3@gmail.com'],
   };
 
-  const bigQueryClientDouble = {
+  const bigQueryClientDouble = (success) => ({
```
Comment (jcamposmk) [inline src/components/Integrations/BigQuery/AuthorizationPage.test.js:13]: can you remove `logRoles` method in AuthorizationPage.test?  It's not used
image

Comment (kapoc3) [reply src/components/Integrations/BigQuery/AuthorizationPage.test.js:13]: suggest accepted and applied

**Comment by jcamposmk** [comment]
:+1:

**Pull request merged**